README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Self-Introduction
+![](https://artificery.com/wp-content/uploads/2017/09/portland-oregon.jpg)
+
+Hi all, my name is Moiz and I'm a graduate student in the MAPSS program. I'm *really* new to coding, but here's my attempt at listing some basic info about myself:
+
+1. Concentration: Political Science
+2. Hometown: Portland, Oregon
+3. Undergraduate Education: Vassar College
+
```

Comment:
Succesful deployment of list syntax in markdown. Also works great thematically with the introduction -- a clear and cogent way of presenting relevant information without being verbose.